client/src/pages/private/PSITable.js

```diff
@@ -0,0 +1,228 @@
+  };
+  return (
+    <>
+      <Dialog
+        title={`Create New Institute`}
+        open={activeModalForm != null}
+        onClose={defaultOnClose}
+      >
+        {activeModalForm === 'new-psi' && (
+          <NewPSIForm
+            initialValues={{
+              instituteName: '',
+              healthAuthority: '',
+              streetAddress: '',
+              city: '',
+              postalCode: '',
+            }}
+            onSubmit={(values) => {
+              handlePSICreate({
+                instituteName: values.instituteName,
+                healthAuthority: values.healthAuthority,
+                streetAddress: values.streetAddress,
+                city: values.city,
+                postalCode: values.postalCode,
+              });
+            }}
+            onClose={defaultOnClose}
+          />
+        )}
+      </Dialog>
+      <Grid
+        container
+        alignContent='flex-start'
+        justify='flex-start'
+        alignItems='center'
+        direction='column'
+      >
+        <Grid
+          container
+          alignContent='flex-start'
+          justify='flex-start'
+          alignItems='center'
+          direction='row'
+        >
+          <Grid item>
+            <Box pl={2} pr={2} pt={1}>
+              <Typography variant='body1' gutterBottom>
+                Filter:
+              </Typography>
+            </Box>
+          </Grid>
+          <Grid item>
+            <Box minWidth={180}>
+              <TableFilter
+                onFilter={(filteredRows) => setRows(filteredRows)}
+                values={healthAuthorities}
+                rows={fetchedRows}
+                label='Health Authority'
+                filterField='health_authority'
+              />
+            </Box>
+          </Grid>
+          <CheckPermissions roles={roles} permittedRoles={['ministry_of_health']}>
+            <Grid container item xs={2} style={{ marginLeft: 'auto', marginRight: 20 }}>
+              <Button
+                onClick={async () => {
+                  setActiveModalForm('new-psi');
+                }}
+                size='medium'
+                text='+ Add PSI'
+              />
+            </Grid>
+          </CheckPermissions>
+        </Grid>
+        <Box pt={2} pb={2} pl={2} pr={2} width='100%'>
+          <Table
+            columns={columns}
+            order={order}
+            orderBy={orderBy}
+            onRequestSort={handleRequestSort}
+            rows={sort(rows)}
+            isLoading={isLoadingData}
+            renderCell={(columnId, row) => {
+              // TODO: Make instituteName clickable
+              if (columnId === 'instituteName') return <a href='/'>{row.instituteName}</a>;
+              if (columnId === 'addCohort')
+                return (
+                  <Button
+                    // onClick={() => history.push(Routes.SiteView + `/${row.id}`)}
+                    onClick={() => console.log('TODO: Implement Add Cohort')}
```

Comment:
I have mixed feelings about leaving console.logs and commented out code.
It might be cleaner to have a toast here and TODO comment